operators/multiclusterobservability/pkg/config/resources_map.go

```diff
@@ -391,8 +391,13 @@ const (
 	MetricsCollectorCPULimits    = ""
 	MetricsCollectorMemoryLimits = ""
 
-	MCOACPULimits    = "200m"
-	MCOAMemoryLimits = "512Mi"
+	MCOACPULimits = "200m"
+	// MCOA runs two controllers, the addon-framework controller and a extra watcher
+	// controller that watches for changes in secrets. Running this extra controller increases
+	// the memory usage of MCOA. In https://issues.redhat.com/browse/ACM-13715
+	// the preformance testing showed that value to be around 3Gi.
+	// For now we will set the limit to 5Gi but will probabbly need to be tuned again.
+	MCOAMemoryLimits = "5Gi"
```

Comment:
Do we want to set limits here at all? We don't for other components, to prevent OOMs/crashes atm, and leave it up to the users